Converters/Tekla/Speckle.Converters.TeklaShared/ServiceRegistration.cs

```diff
@@ -28,6 +31,8 @@ public static IServiceCollection AddTeklaConverters(this IServiceCollection serv
       ConverterSettingsStore<TeklaConversionSettings>
     >();
 
+    serviceCollection.AddTransient<ITypedConverter<TSM.BooleanPart, IEnumerable<Base>>, OpeningToSpeckleConverter>();
```

Comment:
AFAIK you do not need to register `ITypedConverter`s explicitly. Some magic (`serviceCollection.AddMatchingInterfacesAsTransient(converterAssembly)`) can handle them. Did you have a problem before with that?